website/docs/faqs/Core/install-python-compatibility.md

```diff
@@ -5,9 +5,21 @@ sidebar_label: 'Python version'
 id: install-python-compatibility
 ---
 
-<VersionBlock firstVersion="1.1">
+<VersionBlock firstVersion="1.4">
 
-The latest version of `dbt-core` is compatible with Python versions 3.7, 3.8, 3.9, and 3.10.
+The latest version of `dbt-core` is compatible with Python versions 3.7, 3.8, 3.9, 3.10, and 3.11.
+
+:::info
+
+Full support of Python 3.11 across adapters is a work in progress since adapter plugins have additional third-party dependencies. As of January 2023, dbt-snowflake does not yet support Python 3.11 (relevant issue in snowflake-connector-python).
```

Comment:
> relevant issue in snowflake-connector-python

Could you add the link from this text to: https://github.com/snowflakedb/snowflake-connector-python/issues/1345